packages/solana/lib/src/programs/token_program/solana_client_ext.dart

```diff
@@ -131,6 +131,7 @@ extension SolanaClientTokenProgram on SolanaClient {
     String? memo,
     SignatureCallback? onSigned,
     Commitment commitment = Commitment.finalized,
+    TokenProgramType tokenProgramType = TokenProgramType.tokenProgram,
```

Comment:
For consistency with `TokenInstruction`, let's call this parameter `tokenProgram`.